tests/msbuild-mac/src/RoslynSmokeTests.cs

```diff
@@ -8,35 +8,31 @@ namespace Xamarin.MMP.Tests
 	[TestFixture]
 	public partial class MMPTests
 	{
-		const string nugetPath = "/Library/Frameworks/Mono.framework/Versions/Current/Commands/nuget";
-
 		public string RoslynTestProjectRoot => Path.Combine (TI.FindSourceDirectory (), "TestProjects/RoslynTestApp/");
 
-		void RestoreRoslynNuget (string projectType)
+		void RestoreRoslynNuget (string projectPath)
 		{
```

Comment:
This logic only worked if you were in certain directories and was wrong, we can just call msbuild /t:restore and let them handle it anyway.